packages/visualizations/addon/templates/components/navi-visualizations/table.hbs

```diff
@@ -5,36 +5,49 @@
 </div>
 <div class="table-widget__horizontal-scroll-container">
   <div class= 'table-widget-container'>
-      {{#sortable-group class='table-header-row' direction='x' onChange='updateColumnOrder' as | group |}}
-        {{#each columns as |column|}}
+      {{#if isEditing}}
```

Comment:
Question: Do we want to not allow sort and reorder while we are editing? Cos in a way, that is also technically editing the visualization. @jkusa thoughts?

Reply:
We talked about using an icon for _moving_ columns. I think we can wait for that.